### PR TITLE
Bug 1177817. Update Fx 38.0.5 firstrun CTA.

### PR DESCRIPTION
**Pull request by jpetto**

### CODE REVIEW

**Comment by jpetto** [comment]
@alexgibson or @sgarrity r?

File: media/js/firefox/australis/fx38_0_5/firstrun.js [hunk 1]
```diff
@@ -14,6 +14,11 @@
...
 
     // 37 sends user to sync in hamburger menu
+
+    // 2015-07 update:
+    // all CTAs open Sync hamburger menu. ctaDirect var only used to
+    // change text displayed on CTA.
+    // https://bugzilla.mozilla.org/show_bug.cgi?id=1177817#c8
     if (funnelcakeId === 37) {
         ctaDirect = false;
     // 38 and 39 send user directly to signup form
```
Comment (alexgibson) [inline media/js/firefox/australis/fx38_0_5/firstrun.js]: Can some of these comments be removed if they're no longer accurate?

**Comment by alexgibson** [comment]
Aside from some of the comments being a little confusing, this works as expected and code-wise it looks good. r+wc :sake:

**Comment by jpetto** [comment]
Cleaned up that confusing conditional.

**Pull request merged**